qubesadmin/vm/__init__.py

```diff
@@ -394,6 +394,23 @@ def klass(self):
             self._klass = super().klass
         return self._klass
 
+    def get_notes(self) -> str:
+        ''' Get qube notes '''
+        try:
+            response = self.qubesd_call(self._method_dest, 'admin.vm.notes.Get')
+        except Exception as exc:
```

Comment:
This is definitely too broad. If anything, maybe just `qubesadmin.exc.QubesDaemonNoResponseError`, but even that I'm not sure. qubesd should return already correct exception on error.